input_parameters.conf

```diff
@@ -41,6 +41,15 @@ nan_conversion: 1
 parallel:  0
 processes: 8
 
+#------------------------------------
+# Coherence masking options: used by process
+# cohmask: 1 = ON, 0 = OFF
+# cohthresh: coherence threshold value
+# cohfiledir: directory containing the coherence files
+cohmask:   0
+cohthresh:  0.6
+cohfiledir: 
```

Comment:
Can you rename constants "cohfiledir" in input_parameters.conf and	"COH_DIR" defined in configh.py to be same. It helps tracing back to the value.  And add a comment by default cohfiledir has a None value or set a default value.

Reply:
I've changed the lookup key to "COH_FILE_DIR"

Reply:
and added a note about OBS_DIR being the default value